tests/album-create.test.js

```diff
@@ -0,0 +1,37 @@
+const { expect } = require("chai");
```

Comment:
All the test files have some keywords highlighted in yellow (on my machine). It isn't affected the code but to get rid of it you need to add `"mocha": true` to the eslintrc.js Underneath the `"es2021": true,` line would be fine.